pkg/controllers/resources/pods/translate/translator_test.go

```diff
@@ -348,6 +348,7 @@ func TestRewriteHostsTranslation(t *testing.T) {
 			},
 			configMod: func(c *config.VirtualClusterConfig) {
 				c.ControlPlane.Advanced.DefaultImageRegistry = "my-registry"
+				c.Sync.ToHost.Pods.RewriteHosts.InitContainer.Image.Registry = "mirror.gcr.io"
```

Comment:
@squ94wk I wonder if this has any implications for the default installation logic, should we mention this in the docs?